src/main/java/org/snomed/snowstorm/core/data/services/BranchMetadataHelper.java

```diff
@@ -25,6 +25,7 @@ public class BranchMetadataHelper {
 	private static final String DISABLE_CONTENT_AUTOMATIONS_TRANSIENT_METADATA_KEY = transientKey("disableContentAutomations");
 	private static final String CREATING_CODE_SYSTEM_VERSION_TRANSIENT_METADATA_KEY = transientKey("creatingCodeSystemVersion");
 	private static final String CLASSIFICATION_COMMIT_TRANSIENT_METADATA_KEY = transientKey("classificationCommit");
+	private static final String MRCM_SKIP_COMMIT_LISTENER_TRANSIENT_METADATA_KEY = transientKey("skipMrcmUpdateService");
```

Comment:
This transient commit flag is used to conditionally skip `MRCMUpdateService::preCommitCompletion`. 

If the `performUpdate` method is called outwith the `preCommitCompletion` method, then the `performUpdate` method will eventually call itself.